integrationTests/DonationRepositoryTest.php

```diff
@@ -230,22 +234,16 @@ public function testPushSalesForcePendingPushesNewDonationButNotVeryNewDonation(
         \assert($simulatedNow instanceof \DateTimeImmutable);
 
         // assert
-        $method = $donationClientProphecy->createOrUpdate(Argument::type(Donation::class));
+        $busDispatchMethod = $busProphecy->dispatch(Argument::type(Envelope::class))
```

Comment:
probably worth making a more specific assertion of what should be dispatched, instead of just any envelop with any possible contents. Might need to be done via a callback, since in an integration test context it might not be realistic to create an equal reference envelope to compare against.

Reply:
Done in 212c74235d9ea1a531081a44888c64e24cedc757